src/lib/tzcode/gen-zone-abbrevs.pl

```diff
@@ -0,0 +1,207 @@
+    UTC => 0,
+    GMT => 0,
+    EST => -5*60, EDT => -4*60,
+    CST => -6*60, CDT => -5*60,
+    MST => -7*60, MDT => -6*60,
+    PST => -8*60, PDT => -7*60,
+);
+
+my %Regions = (
+    Africa      => 'f',
+    America     => 'm',
+    Antarctica  => 'n',
+    Asia        => 's',
+    Atlantic    => 't',
+    Australia   => 'u',
+    Etc         => 'e',
+    Europe      => 'r',
+    Factory     => 'c',
+    Indian      => 'i',
+    Pacific     => 'p',
+);
+
+my %FlagName;
+
+BEGIN {
+    my %flags = (
+        DT_ZONE_UTC       => 0x01,
+        DT_ZONE_RFC       => 0x02,
+        DT_ZONE_MILITARY  => 0x04,
+        DT_ZONE_AMBIGUOUS => 0x08,
+    );
+    require constant; constant->import(\%flags);
+    %FlagName = reverse %flags;
+}
+
+my $zone_abbrevs_file = shift || 'zone-abbrevs.txt';
+my $tzdata_zi_file    = shift || 'main.zi';
+
+# ACT    -05:00          # Acre Time (South America)
+# ACT    +10:30 / +09:30 # Australian Central Time (Australia)
+# ACWST  +08:45          # Australian Central Western Standard Time (Australia)
+
+my $EntryRx = do {
+    my $name   = '[A-Z][A-Za-z]{0,5}';
+    my $offset = '[+-] (?: [01][0-9] | [2][0-3]) [:] [0-5][0-9]';
+    qr< ($name) \s+ ($offset) (?: \s+ / \s+ ($offset))? \s+ #>xo
+};
+
+my %ZoneAbbrevs;
+
+# read zone-abbrevs.txt with definition of all currently known
+# timezone abbreviations.
+sub read_abbrevs_file($) {
+    my $filename = shift;
+    open( my $fh, '<:raw', $filename )
+      or die qq<Could not open '$filename' for reading: '$!'>;
+
+    while (<$fh>) {
+        next if /\A \s* \# /x;
+
+        my ( $name, $offset1, $offset2 ) = m< \A $EntryRx >x
+          or die qq/Could not parse zone entry at line $_: $./;
+
+        my $encoded = encode($name);
+        my $offset  = 0;
+        my $flags   = 0;
+
+        if ($offset2) {
+            $flags = DT_ZONE_AMBIGUOUS;
+        }
+        else {
+            my ( $h, $m ) = split /[:]/, $offset1;
+            $offset = $h * 60 + $m;
+        }
+
+        if ( exists $ZoneAbbrevs{$encoded} ) {
+            $flags |= DT_ZONE_AMBIGUOUS;
+            $offset = 0;
+        }
+
+        if ( exists $Universal{$name} ) {
+            $flags |= DT_ZONE_UTC;
+            $offset = 0;
+        }
+
+        if ( exists $Military{$name} ) {
+            $flags |= DT_ZONE_MILITARY;
+            $offset = $Military{$name};
+        }
+
+        if ( exists $Rfc{$name} ) {
+            $flags |= DT_ZONE_RFC;
+            $offset = $Rfc{$name};
+        }
+
+        $ZoneAbbrevs{$encoded} = [ $flags, $offset, $name ];
+    }
+}
+
+my %ZoneNames;
+
+sub read_zi_file($) {
+    my $filename = shift;
+    open( my $fh, '<:raw', $filename )
+      or die qq<Could not open '$filename' for reading: '$!'>;
+
+    my $zone;
+    my $format;
+    while (<$fh>) {
+        next if /^#/;
+        if (/^Zone\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)/) {
+            $zone   = $1;
+            $format = $4;
+            $ZoneNames{$zone}++ if $format eq 'LMT';
```

Comment:
And, BTW, that's incorrect condition, not covering all zones described in `main.zi` (i.e. it was not picking up `Etc/UTC` zone which had no LMT rule in there. Corrected it a bit, which regenerated timezone list a bit:

```diff
commit 85652f3f20d16f6b6f9ebfa73724b33d02b7cf38
Author: Timur Safin <tsafin@tarantool.org>
Date:   Sun Apr 24 01:22:28 2022 +0300

    temp - correct regexp for timezone names. Updated list

diff --git a/src/lib/tzcode/gen-zone-abbrevs.pl b/src/lib/tzcode/gen-zone-abbrevs.pl
index a67b8f804..7d3d6948a 100755
--- a/src/lib/tzcode/gen-zone-abbrevs.pl
+++ b/src/lib/tzcode/gen-zone-abbrevs.pl
@@ -149,10 +149,9 @@ sub read_zi_file($) {
     my $format;
     while (<$fh>) {
         next if /^#/;
-        if (/^Zone\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)/) {
+        if (/^Zone\s+([A-Za-z\-\_\/]+)\s+/) {
             $zone   = $1;
-            $format = $4;
-            $ZoneNames{$zone}++ if $format eq 'LMT';
+            $ZoneNames{$zone}++ if $zone =~ q|/|;
             next;
         }
     }
diff --git a/src/lib/tzcode/timezones.h b/src/lib/tzcode/timezones.h
index 1f27db637..cd18acc5b 100644
--- a/src/lib/tzcode/timezones.h
+++ b/src/lib/tzcode/timezones.h
@@ -372,327 +372,352 @@ ZONE_UNIQUE( 751, "America/Blanc-Sablon")
 ZONE_UNIQUE( 752, "America/Boa_Vista")
 ZONE_UNIQUE( 753, "America/Bogota")
 ZONE_UNIQUE( 754, "America/Boise")
-ZONE_UNIQUE( 755, "America/Campo_Grande")
-ZONE_UNIQUE( 756, "America/Cancun")
-ZONE_UNIQUE( 757, "America/Caracas")
-ZONE_UNIQUE( 758, "America/Cayenne")
-ZONE_UNIQUE( 759, "America/Cayman")
-ZONE_UNIQUE( 760, "America/Chicago")
-ZONE_UNIQUE( 761, "America/Chihuahua")
-ZONE_UNIQUE( 762, "America/Coral_Harbour")
-ZONE_UNIQUE( 763, "America/Costa_Rica")
-ZONE_UNIQUE( 764, "America/Creston")
-ZONE_UNIQUE( 765, "America/Cuiaba")
-ZONE_UNIQUE( 766, "America/Curacao")
-ZONE_UNIQUE( 767, "America/Danmarkshavn")
-ZONE_UNIQUE( 768, "America/Dawson")
-ZONE_UNIQUE( 769, "America/Dawson_Creek")
-ZONE_UNIQUE( 770, "America/Denver")
-ZONE_UNIQUE( 771, "America/Detroit")
-ZONE_UNIQUE( 772, "America/Dominica")
-ZONE_UNIQUE( 773, "America/Edmonton")
-ZONE_UNIQUE( 774, "America/Eirunepe")
-ZONE_UNIQUE( 775, "America/El_Salvador")
-ZONE_UNIQUE( 776, "America/Ensenada")
-ZONE_UNIQUE( 777, "America/Fort_Nelson")
-ZONE_UNIQUE( 778, "America/Fortaleza")
-ZONE_UNIQUE( 779, "America/Glace_Bay")
-ZONE_UNIQUE( 780, "America/Goose_Bay")
-ZONE_UNIQUE( 781, "America/Grand_Turk")
-ZONE_UNIQUE( 782, "America/Grenada")
-ZONE_UNIQUE( 783, "America/Guadeloupe")
-ZONE_UNIQUE( 784, "America/Guatemala")
-ZONE_UNIQUE( 785, "America/Guayaquil")
-ZONE_UNIQUE( 786, "America/Guyana")
-ZONE_UNIQUE( 787, "America/Halifax")
-ZONE_UNIQUE( 788, "America/Havana")
-ZONE_UNIQUE( 789, "America/Hermosillo")
-ZONE_UNIQUE( 790, "America/Indiana/Indianapolis")
-ZONE_UNIQUE( 791, "America/Indiana/Knox")
-ZONE_UNIQUE( 792, "America/Indiana/Marengo")
-ZONE_UNIQUE( 793, "America/Indiana/Petersburg")
-ZONE_UNIQUE( 794, "America/Indiana/Tell_City")
-ZONE_UNIQUE( 795, "America/Indiana/Vevay")
-ZONE_UNIQUE( 796, "America/Indiana/Vincennes")
-ZONE_UNIQUE( 797, "America/Indiana/Winamac")
-ZONE_UNIQUE( 798, "America/Jamaica")
-ZONE_UNIQUE( 799, "America/Juneau")
-ZONE_UNIQUE( 800, "America/Kentucky/Louisville")
-ZONE_UNIQUE( 801, "America/Kentucky/Monticello")
-ZONE_UNIQUE( 802, "America/La_Paz")
-ZONE_UNIQUE( 803, "America/Lima")
-ZONE_UNIQUE( 804, "America/Los_Angeles")
-ZONE_UNIQUE( 805, "America/Maceio")
-ZONE_UNIQUE( 806, "America/Managua")
-ZONE_UNIQUE( 807, "America/Manaus")
-ZONE_UNIQUE( 808, "America/Martinique")
-ZONE_UNIQUE( 809, "America/Matamoros")
-ZONE_UNIQUE( 810, "America/Mazatlan")
-ZONE_UNIQUE( 811, "America/Menominee")
-ZONE_UNIQUE( 812, "America/Merida")
-ZONE_UNIQUE( 813, "America/Metlakatla")
-ZONE_UNIQUE( 814, "America/Mexico_City")
-ZONE_UNIQUE( 815, "America/Miquelon")
-ZONE_UNIQUE( 816, "America/Moncton")
-ZONE_UNIQUE( 817, "America/Monterrey")
-ZONE_UNIQUE( 818, "America/Montevideo")
-ZONE_UNIQUE( 819, "America/Montreal")
-ZONE_UNIQUE( 820, "America/Montserrat")
-ZONE_UNIQUE( 821, "America/Nassau")
-ZONE_UNIQUE( 822, "America/New_York")
-ZONE_UNIQUE( 823, "America/Nipigon")
-ZONE_UNIQUE( 824, "America/Nome")
-ZONE_UNIQUE( 825, "America/Noronha")
-ZONE_UNIQUE( 826, "America/North_Dakota/Beulah")
-ZONE_UNIQUE( 827, "America/North_Dakota/Center")
-ZONE_UNIQUE( 828, "America/North_Dakota/New_Salem")
-ZONE_UNIQUE( 829, "America/Nuuk")
-ZONE_UNIQUE( 830, "America/Ojinaga")
-ZONE_UNIQUE( 831, "America/Panama")
-ZONE_UNIQUE( 832, "America/Paramaribo")
-ZONE_UNIQUE( 833, "America/Phoenix")
-ZONE_UNIQUE( 834, "America/Port-au-Prince")
-ZONE_UNIQUE( 835, "America/Port_of_Spain")
-ZONE_UNIQUE( 836, "America/Porto_Velho")
-ZONE_UNIQUE( 837, "America/Puerto_Rico")
-ZONE_UNIQUE( 838, "America/Punta_Arenas")
-ZONE_UNIQUE( 839, "America/Rainy_River")
-ZONE_UNIQUE( 840, "America/Recife")
-ZONE_UNIQUE( 841, "America/Regina")
-ZONE_UNIQUE( 842, "America/Rio_Branco")
-ZONE_UNIQUE( 843, "America/Rosario")
-ZONE_UNIQUE( 844, "America/Santarem")
-ZONE_UNIQUE( 845, "America/Santiago")
-ZONE_UNIQUE( 846, "America/Santo_Domingo")
-ZONE_UNIQUE( 847, "America/Sao_Paulo")
-ZONE_UNIQUE( 848, "America/Scoresbysund")
-ZONE_UNIQUE( 849, "America/Sitka")
-ZONE_UNIQUE( 850, "America/St_Johns")
-ZONE_UNIQUE( 851, "America/St_Kitts")
-ZONE_UNIQUE( 852, "America/St_Lucia")
-ZONE_UNIQUE( 853, "America/St_Thomas")
-ZONE_UNIQUE( 854, "America/St_Vincent")
-ZONE_UNIQUE( 855, "America/Swift_Current")
-ZONE_UNIQUE( 856, "America/Tegucigalpa")
-ZONE_UNIQUE( 857, "America/Thule")
-ZONE_UNIQUE( 858, "America/Thunder_Bay")
-ZONE_UNIQUE( 859, "America/Tijuana")
-ZONE_UNIQUE( 860, "America/Toronto")
-ZONE_UNIQUE( 861, "America/Tortola")
-ZONE_UNIQUE( 862, "America/Vancouver")
-ZONE_UNIQUE( 863, "America/Whitehorse")
-ZONE_UNIQUE( 864, "America/Winnipeg")
-ZONE_UNIQUE( 865, "America/Yakutat")
-ZONE_UNIQUE( 866, "Asia/Aden")
-ZONE_UNIQUE( 867, "Asia/Almaty")
-ZONE_UNIQUE( 868, "Asia/Amman")
-ZONE_UNIQUE( 869, "Asia/Anadyr")
-ZONE_UNIQUE( 870, "Asia/Aqtau")
-ZONE_UNIQUE( 871, "Asia/Aqtobe")
-ZONE_UNIQUE( 872, "Asia/Ashgabat")
-ZONE_UNIQUE( 873, "Asia/Atyrau")
-ZONE_UNIQUE( 874, "Asia/Baghdad")
-ZONE_UNIQUE( 875, "Asia/Bahrain")
-ZONE_UNIQUE( 876, "Asia/Baku")
-ZONE_UNIQUE( 877, "Asia/Bangkok")
-ZONE_UNIQUE( 878, "Asia/Barnaul")
-ZONE_UNIQUE( 879, "Asia/Beirut")
-ZONE_UNIQUE( 880, "Asia/Bishkek")
-ZONE_UNIQUE( 881, "Asia/Brunei")
-ZONE_UNIQUE( 882, "Asia/Chita")
-ZONE_UNIQUE( 883, "Asia/Choibalsan")
-ZONE_UNIQUE( 884, "Asia/Chongqing")
-ZONE_UNIQUE( 885, "Asia/Colombo")
-ZONE_UNIQUE( 886, "Asia/Damascus")
-ZONE_UNIQUE( 887, "Asia/Dhaka")
-ZONE_UNIQUE( 888, "Asia/Dili")
-ZONE_UNIQUE( 889, "Asia/Dubai")
-ZONE_UNIQUE( 890, "Asia/Dushanbe")
-ZONE_UNIQUE( 891, "Asia/Famagusta")
-ZONE_UNIQUE( 892, "Asia/Gaza")
-ZONE_UNIQUE( 893, "Asia/Hanoi")
-ZONE_UNIQUE( 894, "Asia/Harbin")
-ZONE_UNIQUE( 895, "Asia/Hebron")
-ZONE_UNIQUE( 896, "Asia/Ho_Chi_Minh")
-ZONE_UNIQUE( 897, "Asia/Hong_Kong")
-ZONE_UNIQUE( 898, "Asia/Hovd")
-ZONE_UNIQUE( 899, "Asia/Irkutsk")
-ZONE_UNIQUE( 900, "Asia/Jakarta")
-ZONE_UNIQUE( 901, "Asia/Jayapura")
-ZONE_UNIQUE( 902, "Asia/Jerusalem")
-ZONE_UNIQUE( 903, "Asia/Kabul")
-ZONE_UNIQUE( 904, "Asia/Kamchatka")
-ZONE_UNIQUE( 905, "Asia/Karachi")
-ZONE_UNIQUE( 906, "Asia/Kashgar")
-ZONE_UNIQUE( 907, "Asia/Kathmandu")
-ZONE_UNIQUE( 908, "Asia/Khandyga")
-ZONE_UNIQUE( 909, "Asia/Kolkata")
-ZONE_UNIQUE( 910, "Asia/Krasnoyarsk")
-ZONE_UNIQUE( 911, "Asia/Kuala_Lumpur")
-ZONE_UNIQUE( 912, "Asia/Kuching")
-ZONE_UNIQUE( 913, "Asia/Kuwait")
-ZONE_UNIQUE( 914, "Asia/Macau")
-ZONE_UNIQUE( 915, "Asia/Magadan")
-ZONE_UNIQUE( 916, "Asia/Makassar")
-ZONE_UNIQUE( 917, "Asia/Manila")
-ZONE_UNIQUE( 918, "Asia/Muscat")
-ZONE_UNIQUE( 919, "Asia/Nicosia")
-ZONE_UNIQUE( 920, "Asia/Novokuznetsk")
-ZONE_UNIQUE( 921, "Asia/Novosibirsk")
-ZONE_UNIQUE( 922, "Asia/Omsk")
-ZONE_UNIQUE( 923, "Asia/Oral")
-ZONE_UNIQUE( 924, "Asia/Phnom_Penh")
-ZONE_UNIQUE( 925, "Asia/Pontianak")
-ZONE_UNIQUE( 926, "Asia/Pyongyang")
-ZONE_UNIQUE( 927, "Asia/Qatar")
-ZONE_UNIQUE( 928, "Asia/Qostanay")
-ZONE_UNIQUE( 929, "Asia/Qyzylorda")
-ZONE_UNIQUE( 930, "Asia/Riyadh")
-ZONE_UNIQUE( 931, "Asia/Sakhalin")
-ZONE_UNIQUE( 932, "Asia/Samarkand")
-ZONE_UNIQUE( 933, "Asia/Seoul")
-ZONE_UNIQUE( 934, "Asia/Shanghai")
-ZONE_UNIQUE( 935, "Asia/Singapore")
-ZONE_UNIQUE( 936, "Asia/Srednekolymsk")
-ZONE_UNIQUE( 937, "Asia/Taipei")
-ZONE_UNIQUE( 938, "Asia/Tashkent")
-ZONE_UNIQUE( 939, "Asia/Tbilisi")
-ZONE_UNIQUE( 940, "Asia/Tehran")
-ZONE_UNIQUE( 941, "Asia/Tel_Aviv")
-ZONE_UNIQUE( 942, "Asia/Thimphu")
-ZONE_UNIQUE( 943, "Asia/Tokyo")
-ZONE_UNIQUE( 944, "Asia/Tomsk")
-ZONE_UNIQUE( 945, "Asia/Ulaanbaatar")
-ZONE_UNIQUE( 946, "Asia/Urumqi")
-ZONE_UNIQUE( 947, "Asia/Ust-Nera")
-ZONE_UNIQUE( 948, "Asia/Vientiane")
-ZONE_UNIQUE( 949, "Asia/Vladivostok")
-ZONE_UNIQUE( 950, "Asia/Yakutsk")
-ZONE_UNIQUE( 951, "Asia/Yangon")
-ZONE_UNIQUE( 952, "Asia/Yekaterinburg")
-ZONE_UNIQUE( 953, "Asia/Yerevan")
-ZONE_UNIQUE( 954, "Atlantic/Azores")
-ZONE_UNIQUE( 955, "Atlantic/Bermuda")
-ZONE_UNIQUE( 956, "Atlantic/Canary")
-ZONE_UNIQUE( 957, "Atlantic/Cape_Verde")
-ZONE_UNIQUE( 958, "Atlantic/Faroe")
-ZONE_UNIQUE( 959, "Atlantic/Madeira")
-ZONE_UNIQUE( 960, "Atlantic/Reykjavik")
-ZONE_UNIQUE( 961, "Atlantic/South_Georgia")
-ZONE_UNIQUE( 962, "Atlantic/St_Helena")
-ZONE_UNIQUE( 963, "Atlantic/Stanley")
-ZONE_UNIQUE( 964, "Australia/Adelaide")
-ZONE_UNIQUE( 965, "Australia/Brisbane")
-ZONE_UNIQUE( 966, "Australia/Broken_Hill")
-ZONE_UNIQUE( 967, "Australia/Currie")
-ZONE_UNIQUE( 968, "Australia/Darwin")
-ZONE_UNIQUE( 969, "Australia/Eucla")
-ZONE_UNIQUE( 970, "Australia/Hobart")
-ZONE_UNIQUE( 971, "Australia/Lindeman")
-ZONE_UNIQUE( 972, "Australia/Lord_Howe")
-ZONE_UNIQUE( 973, "Australia/Melbourne")
-ZONE_UNIQUE( 974, "Australia/Perth")
-ZONE_UNIQUE( 975, "Australia/Sydney")
-ZONE_UNIQUE( 976, "Europe/Amsterdam")
-ZONE_UNIQUE( 977, "Europe/Andorra")
-ZONE_UNIQUE( 978, "Europe/Astrakhan")
-ZONE_UNIQUE( 979, "Europe/Athens")
-ZONE_UNIQUE( 980, "Europe/Belfast")
-ZONE_UNIQUE( 981, "Europe/Belgrade")
-ZONE_UNIQUE( 982, "Europe/Berlin")
-ZONE_UNIQUE( 983, "Europe/Brussels")
-ZONE_UNIQUE( 984, "Europe/Bucharest")
-ZONE_UNIQUE( 985, "Europe/Budapest")
-ZONE_UNIQUE( 986, "Europe/Chisinau")
-ZONE_UNIQUE( 987, "Europe/Copenhagen")
-ZONE_UNIQUE( 988, "Europe/Dublin")
-ZONE_UNIQUE( 989, "Europe/Gibraltar")
-ZONE_UNIQUE( 990, "Europe/Guernsey")
-ZONE_UNIQUE( 991, "Europe/Helsinki")
-ZONE_UNIQUE( 992, "Europe/Isle_of_Man")
-ZONE_UNIQUE( 993, "Europe/Istanbul")
-ZONE_UNIQUE( 994, "Europe/Jersey")
-ZONE_UNIQUE( 995, "Europe/Kaliningrad")
-ZONE_UNIQUE( 996, "Europe/Kiev")
-ZONE_UNIQUE( 997, "Europe/Kirov")
-ZONE_UNIQUE( 998, "Europe/Lisbon")
-ZONE_UNIQUE( 999, "Europe/Ljubljana")
-ZONE_UNIQUE(1000, "Europe/London")
-ZONE_UNIQUE(1001, "Europe/Luxembourg")
-ZONE_UNIQUE(1002, "Europe/Madrid")
-ZONE_UNIQUE(1003, "Europe/Malta")
-ZONE_UNIQUE(1004, "Europe/Minsk")
-ZONE_UNIQUE(1005, "Europe/Monaco")
-ZONE_UNIQUE(1006, "Europe/Moscow")
-ZONE_UNIQUE(1007, "Europe/Oslo")
-ZONE_UNIQUE(1008, "Europe/Paris")
-ZONE_UNIQUE(1009, "Europe/Prague")
-ZONE_UNIQUE(1010, "Europe/Riga")
-ZONE_UNIQUE(1011, "Europe/Rome")
-ZONE_UNIQUE(1012, "Europe/Samara")
-ZONE_UNIQUE(1013, "Europe/Sarajevo")
-ZONE_UNIQUE(1014, "Europe/Saratov")
-ZONE_UNIQUE(1015, "Europe/Simferopol")
-ZONE_UNIQUE(1016, "Europe/Skopje")
-ZONE_UNIQUE(1017, "Europe/Sofia")
-ZONE_UNIQUE(1018, "Europe/Stockholm")
-ZONE_UNIQUE(1019, "Europe/Tallinn")
-ZONE_UNIQUE(1020, "Europe/Tirane")
-ZONE_UNIQUE(1021, "Europe/Tiraspol")
-ZONE_UNIQUE(1022, "Europe/Ulyanovsk")
-ZONE_UNIQUE(1023, "Europe/Uzhgorod")
-ZONE_UNIQUE(1024, "Europe/Vaduz")
-ZONE_UNIQUE(1025, "Europe/Vienna")
-ZONE_UNIQUE(1026, "Europe/Vilnius")
-ZONE_UNIQUE(1027, "Europe/Volgograd")
-ZONE_UNIQUE(1028, "Europe/Warsaw")
-ZONE_UNIQUE(1029, "Europe/Zagreb")
-ZONE_UNIQUE(1030, "Europe/Zaporozhye")
-ZONE_UNIQUE(1031, "Europe/Zurich")
-ZONE_UNIQUE(1032, "Indian/Antananarivo")
-ZONE_UNIQUE(1033, "Indian/Chagos")
-ZONE_UNIQUE(1034, "Indian/Christmas")
-ZONE_UNIQUE(1035, "Indian/Cocos")
-ZONE_UNIQUE(1036, "Indian/Comoro")
-ZONE_UNIQUE(1037, "Indian/Mahe")
-ZONE_UNIQUE(1038, "Indian/Maldives")
-ZONE_UNIQUE(1039, "Indian/Mauritius")
-ZONE_UNIQUE(1040, "Indian/Mayotte")
-ZONE_UNIQUE(1041, "Indian/Reunion")
-ZONE_UNIQUE(1048, "Pacific/Apia")
-ZONE_UNIQUE(1049, "Pacific/Auckland")
-ZONE_UNIQUE(1050, "Pacific/Bougainville")
-ZONE_UNIQUE(1051, "Pacific/Chatham")
-ZONE_UNIQUE(1052, "Pacific/Chuuk")
-ZONE_UNIQUE(1053, "Pacific/Easter")
-ZONE_UNIQUE(1054, "Pacific/Efate")
-ZONE_UNIQUE(1055, "Pacific/Fakaofo")
-ZONE_UNIQUE(1056, "Pacific/Fiji")
-ZONE_UNIQUE(1057, "Pacific/Funafuti")
-ZONE_UNIQUE(1058, "Pacific/Galapagos")
-ZONE_UNIQUE(1059, "Pacific/Gambier")
-ZONE_UNIQUE(1060, "Pacific/Guadalcanal")
-ZONE_UNIQUE(1061, "Pacific/Guam")
-ZONE_UNIQUE(1062, "Pacific/Honolulu")
-ZONE_UNIQUE(1063, "Pacific/Kiritimati")
-ZONE_UNIQUE(1064, "Pacific/Kosrae")
-ZONE_UNIQUE(1065, "Pacific/Kwajalein")
-ZONE_UNIQUE(1066, "Pacific/Majuro")
-ZONE_UNIQUE(1067, "Pacific/Marquesas")
-ZONE_UNIQUE(1068, "Pacific/Midway")
-ZONE_UNIQUE(1069, "Pacific/Nauru")
-ZONE_UNIQUE(1070, "Pacific/Niue")
-ZONE_UNIQUE(1071, "Pacific/Norfolk")
-ZONE_UNIQUE(1072, "Pacific/Noumea")
-ZONE_UNIQUE(1073, "Pacific/Pago_Pago")
-ZONE_UNIQUE(1074, "Pacific/Palau")
-ZONE_UNIQUE(1075, "Pacific/Pitcairn")
-ZONE_UNIQUE(1076, "Pacific/Pohnpei")
-ZONE_UNIQUE(1077, "Pacific/Port_Moresby")
-ZONE_UNIQUE(1078, "Pacific/Rarotonga")
-ZONE_UNIQUE(1079, "Pacific/Saipan")
-ZONE_UNIQUE(1080, "Pacific/Tahiti")
-ZONE_UNIQUE(1081, "Pacific/Tarawa")
-ZONE_UNIQUE(1082, "Pacific/Tongatapu")
-ZONE_UNIQUE(1083, "Pacific/Wake")
-ZONE_UNIQUE(1084, "Pacific/Wallis")
+ZONE_UNIQUE( 755, "America/Cambridge_Bay")
+ZONE_UNIQUE( 756, "America/Campo_Grande")
+ZONE_UNIQUE( 757, "America/Cancun")
+ZONE_UNIQUE( 758, "America/Caracas")
+ZONE_UNIQUE( 759, "America/Cayenne")
+ZONE_UNIQUE( 760, "America/Cayman")
+ZONE_UNIQUE( 761, "America/Chicago")
+ZONE_UNIQUE( 762, "America/Chihuahua")
+ZONE_UNIQUE( 763, "America/Coral_Harbour")
+ZONE_UNIQUE( 764, "America/Costa_Rica")
+ZONE_UNIQUE( 765, "America/Creston")
+ZONE_UNIQUE( 766, "America/Cuiaba")
+ZONE_UNIQUE( 767, "America/Curacao")
+ZONE_UNIQUE( 768, "America/Danmarkshavn")
+ZONE_UNIQUE( 769, "America/Dawson")
+ZONE_UNIQUE( 770, "America/Dawson_Creek")
+ZONE_UNIQUE( 771, "America/Denver")
+ZONE_UNIQUE( 772, "America/Detroit")
+ZONE_UNIQUE( 773, "America/Dominica")
+ZONE_UNIQUE( 774, "America/Edmonton")
+ZONE_UNIQUE( 775, "America/Eirunepe")
+ZONE_UNIQUE( 776, "America/El_Salvador")
+ZONE_UNIQUE( 777, "America/Ensenada")
+ZONE_UNIQUE( 778, "America/Fort_Nelson")
+ZONE_UNIQUE( 779, "America/Fortaleza")
+ZONE_UNIQUE( 780, "America/Glace_Bay")
+ZONE_UNIQUE( 781, "America/Goose_Bay")
+ZONE_UNIQUE( 782, "America/Grand_Turk")
+ZONE_UNIQUE( 783, "America/Grenada")
+ZONE_UNIQUE( 784, "America/Guadeloupe")
+ZONE_UNIQUE( 785, "America/Guatemala")
+ZONE_UNIQUE( 786, "America/Guayaquil")
+ZONE_UNIQUE( 787, "America/Guyana")
+ZONE_UNIQUE( 788, "America/Halifax")
+ZONE_UNIQUE( 789, "America/Havana")
+ZONE_UNIQUE( 790, "America/Hermosillo")
+ZONE_UNIQUE( 791, "America/Indiana/Indianapolis")
+ZONE_UNIQUE( 792, "America/Indiana/Knox")
+ZONE_UNIQUE( 793, "America/Indiana/Marengo")
+ZONE_UNIQUE( 794, "America/Indiana/Petersburg")
+ZONE_UNIQUE( 795, "America/Indiana/Tell_City")
+ZONE_UNIQUE( 796, "America/Indiana/Vevay")
+ZONE_UNIQUE( 797, "America/Indiana/Vincennes")
+ZONE_UNIQUE( 798, "America/Indiana/Winamac")
+ZONE_UNIQUE( 799, "America/Inuvik")
+ZONE_UNIQUE( 800, "America/Iqaluit")
+ZONE_UNIQUE( 801, "America/Jamaica")
+ZONE_UNIQUE( 802, "America/Juneau")
+ZONE_UNIQUE( 803, "America/Kentucky/Louisville")
+ZONE_UNIQUE( 804, "America/Kentucky/Monticello")
+ZONE_UNIQUE( 805, "America/La_Paz")
+ZONE_UNIQUE( 806, "America/Lima")
+ZONE_UNIQUE( 807, "America/Los_Angeles")
+ZONE_UNIQUE( 808, "America/Maceio")
+ZONE_UNIQUE( 809, "America/Managua")
+ZONE_UNIQUE( 810, "America/Manaus")
+ZONE_UNIQUE( 811, "America/Martinique")
+ZONE_UNIQUE( 812, "America/Matamoros")
+ZONE_UNIQUE( 813, "America/Mazatlan")
+ZONE_UNIQUE( 814, "America/Menominee")
+ZONE_UNIQUE( 815, "America/Merida")
+ZONE_UNIQUE( 816, "America/Metlakatla")
+ZONE_UNIQUE( 817, "America/Mexico_City")
+ZONE_UNIQUE( 818, "America/Miquelon")
+ZONE_UNIQUE( 819, "America/Moncton")
+ZONE_UNIQUE( 820, "America/Monterrey")
+ZONE_UNIQUE( 821, "America/Montevideo")
+ZONE_UNIQUE( 822, "America/Montreal")
+ZONE_UNIQUE( 823, "America/Montserrat")
+ZONE_UNIQUE( 824, "America/Nassau")
+ZONE_UNIQUE( 825, "America/New_York")
+ZONE_UNIQUE( 826, "America/Nipigon")
+ZONE_UNIQUE( 827, "America/Nome")
+ZONE_UNIQUE( 828, "America/Noronha")
+ZONE_UNIQUE( 829, "America/North_Dakota/Beulah")
+ZONE_UNIQUE( 830, "America/North_Dakota/Center")
+ZONE_UNIQUE( 831, "America/North_Dakota/New_Salem")
+ZONE_UNIQUE( 832, "America/Nuuk")
+ZONE_UNIQUE( 833, "America/Ojinaga")
+ZONE_UNIQUE( 834, "America/Panama")
+ZONE_UNIQUE( 835, "America/Pangnirtung")
+ZONE_UNIQUE( 836, "America/Paramaribo")
+ZONE_UNIQUE( 837, "America/Phoenix")
+ZONE_UNIQUE( 838, "America/Port-au-Prince")
+ZONE_UNIQUE( 839, "America/Port_of_Spain")
+ZONE_UNIQUE( 840, "America/Porto_Velho")
+ZONE_UNIQUE( 841, "America/Puerto_Rico")
+ZONE_UNIQUE( 842, "America/Punta_Arenas")
+ZONE_UNIQUE( 843, "America/Rainy_River")
+ZONE_UNIQUE( 844, "America/Rankin_Inlet")
+ZONE_UNIQUE( 845, "America/Recife")
+ZONE_UNIQUE( 846, "America/Regina")
+ZONE_UNIQUE( 847, "America/Resolute")
+ZONE_UNIQUE( 848, "America/Rio_Branco")
+ZONE_UNIQUE( 849, "America/Rosario")
+ZONE_UNIQUE( 850, "America/Santarem")
+ZONE_UNIQUE( 851, "America/Santiago")
+ZONE_UNIQUE( 852, "America/Santo_Domingo")
+ZONE_UNIQUE( 853, "America/Sao_Paulo")
+ZONE_UNIQUE( 854, "America/Scoresbysund")
+ZONE_UNIQUE( 855, "America/Sitka")
+ZONE_UNIQUE( 856, "America/St_Johns")
+ZONE_UNIQUE( 857, "America/St_Kitts")
+ZONE_UNIQUE( 858, "America/St_Lucia")
+ZONE_UNIQUE( 859, "America/St_Thomas")
+ZONE_UNIQUE( 860, "America/St_Vincent")
+ZONE_UNIQUE( 861, "America/Swift_Current")
+ZONE_UNIQUE( 862, "America/Tegucigalpa")
+ZONE_UNIQUE( 863, "America/Thule")
+ZONE_UNIQUE( 864, "America/Thunder_Bay")
+ZONE_UNIQUE( 865, "America/Tijuana")
+ZONE_UNIQUE( 866, "America/Toronto")
+ZONE_UNIQUE( 867, "America/Tortola")
+ZONE_UNIQUE( 868, "America/Vancouver")
+ZONE_UNIQUE( 869, "America/Whitehorse")
+ZONE_UNIQUE( 870, "America/Winnipeg")
+ZONE_UNIQUE( 871, "America/Yakutat")
+ZONE_UNIQUE( 872, "America/Yellowknife")
+ZONE_UNIQUE( 873, "Antarctica/Casey")
+ZONE_UNIQUE( 874, "Antarctica/Davis")
+ZONE_UNIQUE( 875, "Antarctica/DumontDUrville")
+ZONE_UNIQUE( 876, "Antarctica/Macquarie")
+ZONE_UNIQUE( 877, "Antarctica/Mawson")
+ZONE_UNIQUE( 878, "Antarctica/McMurdo")
+ZONE_UNIQUE( 879, "Antarctica/Palmer")
+ZONE_UNIQUE( 880, "Antarctica/Rothera")
+ZONE_UNIQUE( 881, "Antarctica/Syowa")
+ZONE_UNIQUE( 882, "Antarctica/Troll")
+ZONE_UNIQUE( 883, "Antarctica/Vostok")
+ZONE_UNIQUE( 884, "Asia/Aden")
+ZONE_UNIQUE( 885, "Asia/Almaty")
+ZONE_UNIQUE( 886, "Asia/Amman")
+ZONE_UNIQUE( 887, "Asia/Anadyr")
+ZONE_UNIQUE( 888, "Asia/Aqtau")
+ZONE_UNIQUE( 889, "Asia/Aqtobe")
+ZONE_UNIQUE( 890, "Asia/Ashgabat")
+ZONE_UNIQUE( 891, "Asia/Atyrau")
+ZONE_UNIQUE( 892, "Asia/Baghdad")
+ZONE_UNIQUE( 893, "Asia/Bahrain")
+ZONE_UNIQUE( 894, "Asia/Baku")
+ZONE_UNIQUE( 895, "Asia/Bangkok")
+ZONE_UNIQUE( 896, "Asia/Barnaul")
+ZONE_UNIQUE( 897, "Asia/Beirut")
+ZONE_UNIQUE( 898, "Asia/Bishkek")
+ZONE_UNIQUE( 899, "Asia/Brunei")
+ZONE_UNIQUE( 900, "Asia/Chita")
+ZONE_UNIQUE( 901, "Asia/Choibalsan")
+ZONE_UNIQUE( 902, "Asia/Chongqing")
+ZONE_UNIQUE( 903, "Asia/Colombo")
+ZONE_UNIQUE( 904, "Asia/Damascus")
+ZONE_UNIQUE( 905, "Asia/Dhaka")
+ZONE_UNIQUE( 906, "Asia/Dili")
+ZONE_UNIQUE( 907, "Asia/Dubai")
+ZONE_UNIQUE( 908, "Asia/Dushanbe")
+ZONE_UNIQUE( 909, "Asia/Famagusta")
+ZONE_UNIQUE( 910, "Asia/Gaza")
+ZONE_UNIQUE( 911, "Asia/Hanoi")
+ZONE_UNIQUE( 912, "Asia/Harbin")
+ZONE_UNIQUE( 913, "Asia/Hebron")
+ZONE_UNIQUE( 914, "Asia/Ho_Chi_Minh")
+ZONE_UNIQUE( 915, "Asia/Hong_Kong")
+ZONE_UNIQUE( 916, "Asia/Hovd")
+ZONE_UNIQUE( 917, "Asia/Irkutsk")
+ZONE_UNIQUE( 918, "Asia/Jakarta")
+ZONE_UNIQUE( 919, "Asia/Jayapura")
+ZONE_UNIQUE( 920, "Asia/Jerusalem")
+ZONE_UNIQUE( 921, "Asia/Kabul")
+ZONE_UNIQUE( 922, "Asia/Kamchatka")
+ZONE_UNIQUE( 923, "Asia/Karachi")
+ZONE_UNIQUE( 924, "Asia/Kashgar")
+ZONE_UNIQUE( 925, "Asia/Kathmandu")
+ZONE_UNIQUE( 926, "Asia/Khandyga")
+ZONE_UNIQUE( 927, "Asia/Kolkata")
+ZONE_UNIQUE( 928, "Asia/Krasnoyarsk")
+ZONE_UNIQUE( 929, "Asia/Kuala_Lumpur")
+ZONE_UNIQUE( 930, "Asia/Kuching")
+ZONE_UNIQUE( 931, "Asia/Kuwait")
+ZONE_UNIQUE( 932, "Asia/Macau")
+ZONE_UNIQUE( 933, "Asia/Magadan")
+ZONE_UNIQUE( 934, "Asia/Makassar")
+ZONE_UNIQUE( 935, "Asia/Manila")
+ZONE_UNIQUE( 936, "Asia/Muscat")
+ZONE_UNIQUE( 937, "Asia/Nicosia")
+ZONE_UNIQUE( 938, "Asia/Novokuznetsk")
+ZONE_UNIQUE( 939, "Asia/Novosibirsk")
+ZONE_UNIQUE( 940, "Asia/Omsk")
+ZONE_UNIQUE( 941, "Asia/Oral")
+ZONE_UNIQUE( 942, "Asia/Phnom_Penh")
+ZONE_UNIQUE( 943, "Asia/Pontianak")
+ZONE_UNIQUE( 944, "Asia/Pyongyang")
+ZONE_UNIQUE( 945, "Asia/Qatar")
+ZONE_UNIQUE( 946, "Asia/Qostanay")
+ZONE_UNIQUE( 947, "Asia/Qyzylorda")
+ZONE_UNIQUE( 948, "Asia/Riyadh")
+ZONE_UNIQUE( 949, "Asia/Sakhalin")
+ZONE_UNIQUE( 950, "Asia/Samarkand")
+ZONE_UNIQUE( 951, "Asia/Seoul")
+ZONE_UNIQUE( 952, "Asia/Shanghai")
+ZONE_UNIQUE( 953, "Asia/Singapore")
+ZONE_UNIQUE( 954, "Asia/Srednekolymsk")
+ZONE_UNIQUE( 955, "Asia/Taipei")
+ZONE_UNIQUE( 956, "Asia/Tashkent")
+ZONE_UNIQUE( 957, "Asia/Tbilisi")
+ZONE_UNIQUE( 958, "Asia/Tehran")
+ZONE_UNIQUE( 959, "Asia/Tel_Aviv")
+ZONE_UNIQUE( 960, "Asia/Thimphu")
+ZONE_UNIQUE( 961, "Asia/Tokyo")
+ZONE_UNIQUE( 962, "Asia/Tomsk")
+ZONE_UNIQUE( 963, "Asia/Ulaanbaatar")
+ZONE_UNIQUE( 964, "Asia/Urumqi")
+ZONE_UNIQUE( 965, "Asia/Ust-Nera")
+ZONE_UNIQUE( 966, "Asia/Vientiane")
+ZONE_UNIQUE( 967, "Asia/Vladivostok")
+ZONE_UNIQUE( 968, "Asia/Yakutsk")
+ZONE_UNIQUE( 969, "Asia/Yangon")
+ZONE_UNIQUE( 970, "Asia/Yekaterinburg")
+ZONE_UNIQUE( 971, "Asia/Yerevan")
+ZONE_UNIQUE( 972, "Atlantic/Azores")
+ZONE_UNIQUE( 973, "Atlantic/Bermuda")
+ZONE_UNIQUE( 974, "Atlantic/Canary")
+ZONE_UNIQUE( 975, "Atlantic/Cape_Verde")
+ZONE_UNIQUE( 976, "Atlantic/Faroe")
+ZONE_UNIQUE( 977, "Atlantic/Jan_Mayen")
+ZONE_UNIQUE( 978, "Atlantic/Madeira")
+ZONE_UNIQUE( 979, "Atlantic/Reykjavik")
+ZONE_UNIQUE( 980, "Atlantic/South_Georgia")
+ZONE_UNIQUE( 981, "Atlantic/St_Helena")
+ZONE_UNIQUE( 982, "Atlantic/Stanley")
+ZONE_UNIQUE( 983, "Australia/Adelaide")
+ZONE_UNIQUE( 984, "Australia/Brisbane")
+ZONE_UNIQUE( 985, "Australia/Broken_Hill")
+ZONE_UNIQUE( 986, "Australia/Currie")
+ZONE_UNIQUE( 987, "Australia/Darwin")
+ZONE_UNIQUE( 988, "Australia/Eucla")
+ZONE_UNIQUE( 989, "Australia/Hobart")
+ZONE_UNIQUE( 990, "Australia/Lindeman")
+ZONE_UNIQUE( 991, "Australia/Lord_Howe")
+ZONE_UNIQUE( 992, "Australia/Melbourne")
+ZONE_UNIQUE( 993, "Australia/Perth")
+ZONE_UNIQUE( 994, "Australia/Sydney")
+ZONE_UNIQUE(1000, "Etc/GMT")
+ZONE_UNIQUE(1001, "Etc/UTC")
+ZONE_UNIQUE(1002, "Europe/Amsterdam")
+ZONE_UNIQUE(1003, "Europe/Andorra")
+ZONE_UNIQUE(1004, "Europe/Astrakhan")
+ZONE_UNIQUE(1005, "Europe/Athens")
+ZONE_UNIQUE(1006, "Europe/Belfast")
+ZONE_UNIQUE(1007, "Europe/Belgrade")
+ZONE_UNIQUE(1008, "Europe/Berlin")
+ZONE_UNIQUE(1009, "Europe/Brussels")
+ZONE_UNIQUE(1010, "Europe/Bucharest")
+ZONE_UNIQUE(1011, "Europe/Budapest")
+ZONE_UNIQUE(1012, "Europe/Chisinau")
+ZONE_UNIQUE(1013, "Europe/Copenhagen")
+ZONE_UNIQUE(1014, "Europe/Dublin")
+ZONE_UNIQUE(1015, "Europe/Gibraltar")
+ZONE_UNIQUE(1016, "Europe/Guernsey")
+ZONE_UNIQUE(1017, "Europe/Helsinki")
+ZONE_UNIQUE(1018, "Europe/Isle_of_Man")
+ZONE_UNIQUE(1019, "Europe/Istanbul")
+ZONE_UNIQUE(1020, "Europe/Jersey")
+ZONE_UNIQUE(1021, "Europe/Kaliningrad")
+ZONE_UNIQUE(1022, "Europe/Kiev")
+ZONE_UNIQUE(1023, "Europe/Kirov")
+ZONE_UNIQUE(1024, "Europe/Lisbon")
+ZONE_UNIQUE(1025, "Europe/Ljubljana")
+ZONE_UNIQUE(1026, "Europe/London")
+ZONE_UNIQUE(1027, "Europe/Luxembourg")
+ZONE_UNIQUE(1028, "Europe/Madrid")
+ZONE_UNIQUE(1029, "Europe/Malta")
+ZONE_UNIQUE(1030, "Europe/Minsk")
+ZONE_UNIQUE(1031, "Europe/Monaco")
+ZONE_UNIQUE(1032, "Europe/Moscow")
+ZONE_UNIQUE(1033, "Europe/Oslo")
+ZONE_UNIQUE(1034, "Europe/Paris")
+ZONE_UNIQUE(1035, "Europe/Prague")
+ZONE_UNIQUE(1036, "Europe/Riga")
+ZONE_UNIQUE(1037, "Europe/Rome")
+ZONE_UNIQUE(1038, "Europe/Samara")
+ZONE_UNIQUE(1039, "Europe/Sarajevo")
+ZONE_UNIQUE(1040, "Europe/Saratov")
+ZONE_UNIQUE(1041, "Europe/Simferopol")
+ZONE_UNIQUE(1042, "Europe/Skopje")
+ZONE_UNIQUE(1043, "Europe/Sofia")
+ZONE_UNIQUE(1044, "Europe/Stockholm")
+ZONE_UNIQUE(1045, "Europe/Tallinn")
+ZONE_UNIQUE(1046, "Europe/Tirane")
+ZONE_UNIQUE(1047, "Europe/Tiraspol")
+ZONE_UNIQUE(1048, "Europe/Ulyanovsk")
+ZONE_UNIQUE(1049, "Europe/Uzhgorod")
+ZONE_UNIQUE(1050, "Europe/Vaduz")
+ZONE_UNIQUE(1051, "Europe/Vienna")
+ZONE_UNIQUE(1052, "Europe/Vilnius")
+ZONE_UNIQUE(1053, "Europe/Volgograd")
+ZONE_UNIQUE(1054, "Europe/Warsaw")
+ZONE_UNIQUE(1055, "Europe/Zagreb")
+ZONE_UNIQUE(1056, "Europe/Zaporozhye")
+ZONE_UNIQUE(1057, "Europe/Zurich")
+ZONE_UNIQUE(1064, "Indian/Antananarivo")
+ZONE_UNIQUE(1065, "Indian/Chagos")
+ZONE_UNIQUE(1066, "Indian/Christmas")
+ZONE_UNIQUE(1067, "Indian/Cocos")
+ZONE_UNIQUE(1068, "Indian/Comoro")
+ZONE_UNIQUE(1069, "Indian/Kerguelen")
+ZONE_UNIQUE(1070, "Indian/Mahe")
+ZONE_UNIQUE(1071, "Indian/Maldives")
+ZONE_UNIQUE(1072, "Indian/Mauritius")
+ZONE_UNIQUE(1073, "Indian/Mayotte")
+ZONE_UNIQUE(1074, "Indian/Reunion")
+ZONE_UNIQUE(1080, "Pacific/Apia")
+ZONE_UNIQUE(1081, "Pacific/Auckland")
+ZONE_UNIQUE(1082, "Pacific/Bougainville")
+ZONE_UNIQUE(1083, "Pacific/Chatham")
+ZONE_UNIQUE(1084, "Pacific/Chuuk")
+ZONE_UNIQUE(1085, "Pacific/Easter")
+ZONE_UNIQUE(1086, "Pacific/Efate")
+ZONE_UNIQUE(1087, "Pacific/Enderbury")
+ZONE_UNIQUE(1088, "Pacific/Fakaofo")
+ZONE_UNIQUE(1089, "Pacific/Fiji")
+ZONE_UNIQUE(1090, "Pacific/Funafuti")
+ZONE_UNIQUE(1091, "Pacific/Galapagos")
+ZONE_UNIQUE(1092, "Pacific/Gambier")
+ZONE_UNIQUE(1093, "Pacific/Guadalcanal")
+ZONE_UNIQUE(1094, "Pacific/Guam")
+ZONE_UNIQUE(1095, "Pacific/Honolulu")
+ZONE_UNIQUE(1096, "Pacific/Johnston")
+ZONE_UNIQUE(1097, "Pacific/Kanton")
+ZONE_UNIQUE(1098, "Pacific/Kiritimati")
+ZONE_UNIQUE(1099, "Pacific/Kosrae")
+ZONE_UNIQUE(1100, "Pacific/Kwajalein")
+ZONE_UNIQUE(1101, "Pacific/Majuro")
+ZONE_UNIQUE(1102, "Pacific/Marquesas")
+ZONE_UNIQUE(1103, "Pacific/Midway")
+ZONE_UNIQUE(1104, "Pacific/Nauru")
+ZONE_UNIQUE(1105, "Pacific/Niue")
+ZONE_UNIQUE(1106, "Pacific/Norfolk")
+ZONE_UNIQUE(1107, "Pacific/Noumea")
+ZONE_UNIQUE(1108, "Pacific/Pago_Pago")
+ZONE_UNIQUE(1109, "Pacific/Palau")
+ZONE_UNIQUE(1110, "Pacific/Pitcairn")
+ZONE_UNIQUE(1111, "Pacific/Pohnpei")
+ZONE_UNIQUE(1112, "Pacific/Port_Moresby")
+ZONE_UNIQUE(1113, "Pacific/Rarotonga")
+ZONE_UNIQUE(1114, "Pacific/Saipan")
+ZONE_UNIQUE(1115, "Pacific/Tahiti")
+ZONE_UNIQUE(1116, "Pacific/Tarawa")
+ZONE_UNIQUE(1117, "Pacific/Tongatapu")
+ZONE_UNIQUE(1118, "Pacific/Wake")
+ZONE_UNIQUE(1119, "Pacific/Wallis")
```